app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -214,6 +222,17 @@ private class HomeTileAdapter(
         }
     }
 
+    fun setLayoutParams(iconView: View, tileMarginValue: Int, tileHeight: Int) {
+        val marginLayoutParams = iconView.layoutParams as ViewGroup.MarginLayoutParams
+        val marginValue = iconView.resources.getDimensionPixelSize(tileMarginValue)
+        marginLayoutParams.setMargins(marginValue, marginValue, marginValue, marginValue)
+        val layoutParams = iconView.layoutParams as LinearLayout.LayoutParams
+        layoutParams.weight = 1f
+        layoutParams.width = LinearLayout.LayoutParams.MATCH_PARENT
+        layoutParams.height = iconView.resources.getDimensionPixelSize(tileHeight)
```

Comment:
Setting an explicit height isn't necessary: weight is setting the height for us. We only need to update the margins